pkg/apps/postgres.go

```diff
@@ -54,6 +55,7 @@ type Postgres struct {
 	UserPasswordSecretKey  string
 	AdminPasswordSecretKey string
 	EnvConfigMapRefs       []string
+	Config                 *protoform.Config
```

Comment:
I'd add a TerminationGracePeriodSeconds field here instead of directly passing *protoform.Config

Reply:
> I'd add a TerminationGracePeriodSeconds field here instead of directly passing *protoform.Config

Hmm..I'm using protoform.Config because it can be set by synopsys-ctl

Reply:
Never mind, I think I got what you mean, just updated the PR